spark/interpreter/src/main/java/org/apache/zeppelin/spark/SparkRInterpreter.java

```diff
@@ -101,37 +104,28 @@ public void open() throws InterpreterException {
     try {
       zeppelinR.open();
     } catch (IOException e) {
-      logger.error("Exception while opening SparkRInterpreter", e);
-      throw new InterpreterException(e);
+      throw new InterpreterException("Exception while opening SparkRInterpreter", e);
     }
 
     if (useKnitr()) {
       zeppelinR.eval("library('knitr')");
     }
-    renderOptions = getProperty("zeppelin.R.render.options");
-  }
-
-  String getJobGroup(InterpreterContext context){
-    return "zeppelin-" + context.getParagraphId();
+    renderOptions = getProperty("zeppelin.R.render.options",
+        "out.format = 'html', comment = NA, echo = FALSE, results = 'asis', message = F, " +
```

Comment:
how do we find this default value?

Reply:
The default value is in interpreter-setting.json. It it better to load default properties instead of specify it explicitly like this (there's many such code in zeppelin). This could be done in another PR in future.